desktop/cmp/rest/RestGridModel.js

```diff
@@ -174,60 +172,52 @@ export class RestGridModel {
         this.formModel.openClone(clone);
     }
 
-    @action
-    deleteSelection() {
-        const record = this.selectedRecord;
-        if (record) this.deleteRecord(record);
-    }
-
-    @action
-    deleteRecord(record) {
+    async deleteRecord(record) {
         throwIf(this.readonly, 'Record not deleted: this grid is read-only');
         this.store.deleteRecordAsync(record)
             .then(() => this.formModel.close())
             .catchDefault();
     }
 
-    @action
-    editSelection() {
-        const record = this.selectedRecord;
-        if (record) this.editRecord(record);
+    async bulkDeleteRecordsAsync(records) {
+        throwIf(this.readonly, 'Records not deleted: this grid is read-only');
+        const resp = await this.store.bulkDeleteRecordsAsync(records).catchDefault(),
+            intent = resp.fail > 0 ? 'warning' : 'success',
```

Comment:
don't think this does what you want if the server throws.  Gonna still toast some nonesense, even if exception is thrownt.

Reply:
Thanks again. Need to learn to think more about the unhappy path.

Reply:
sorry to be a pain, but for symmetry's sake if deleteRecord is catching the exception, bulkDeleteRecordsAsync should be catching the exception.  Also, let's make the method `deleteRecordAsync` to follow our conventions and for symmetry.